src/content/en/2022/capabilities.md

```diff
@@ -1,13 +1,13 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Capabilities
-#TODO - Review and update chapter description
 description: Capabilities chapter of the 2022 Web Almanac covering brand-new, powerful web platform APIs that give web apps access to hardware interfaces, enhance web-based productivity apps, and more.
 authors: [MichaelSolati]
 reviewers: [webmaxru, hemanth, beaufortfrancois, tomayac, christianliebel]
-analysts: [jrstanley]
-editors: []
+analysts: [tunetheweb]
+editors: [tunetheweb]
 translators: []
+MichaelSolati_bio: Technically sophisticated Developer Advocate with a dynamic collection of skills and experience. Talent for efficiently and creatively designing web-based applications utilizing modern development technologies, tools, and methodologies. Passion for championing the development of developers and customer skills/knowledge through expert training abilities and tools. Proven track record of successfully delivering optimized results for strategic technical projects and initiatives.
 results: https://docs.google.com/spreadsheets/d/13S9FRj8OPRtoMPb94jFh6pPNz3lNS9yztIaorZYe288/
 featured_quote: TODO
 featured_stat_1: TODO
```

Comment:
@MichaelSolati can you pull one quote, and three stats, which will be randomly displayed on the home page to promote this chapter.

Here's the examples from last year's chapter:

```yaml
featured_quote: Capabilities are new web platform APIs that unlock entirely new use cases for web applications.
featured_stat_1: 9%
featured_stat_label_1: Desktop web sites using the _Web Share API_
featured_stat_2: 8.25%
featured_stat_label_2: Mobile sites using the _Async Clipboard API_
featured_stat_3: 11
featured_stat_label_3: Mobile sites using _Declarative Link Capturing_
```

Which displayed like this:

<img width="606" alt="image" src="https://user-images.githubusercontent.com/10931297/191618268-d2a3cd0a-1a26-4113-bdd7-38aa7af35836.png">